### DISCUSSION TITLE
How to withdraw/send out btc

### DISCUSSION THREAD

**Discussion by DarkPID** [btcpayserver/btcpayserver#3988] [Technical Support] [answered]
Hi,
So I have the plugin all set up and have been receiving payments with no issues. However, I can't seem to withdraw the btc or send out to a btc address. When attempting to, I get prompted to "sign transaction" which leaves me clueless on what to do. Is that the only way to send out?

**Comment by pavlenex** [comment, marked as answer]
Yes, to send a transaction out of your wallet, you need to sign it, just like with any other wallet.
It depends on how you configured your btcpay wallet https://docs.btcpayserver.org/WalletSetup/
And accordingly then follow the wizard  https://docs.btcpayserver.org/Wallet/

**Reply by DarkPID** [reply]
Thank you for the reply. 
So when I try to sign the transaction so I can send, I get 3 options. The hardware wallet (which I do not have), PSBT and private key or seed. I try to do the private key or seed option but after entering my seed, I get this error "The master fingerprint and/or account key path of your seed are not set in the wallet settings.". 
Is there any easier way to send out btc?

**Reply by bolatovumar** [reply]
@DarkPID go to you wallet settings in BTCPay under `/onchain/BTC/settings` and make sure you have the "Master fingerprint" and "Account key path" fields filled out. If you don't have one of these fields filled out you will get this error (exactly like it says in the error message).

Here are example values from Regtest:

![Capture](https://user-images.githubusercontent.com/1934678/185552539-d2384b77-ff1d-418a-b7a2-f6efd7c4f86f.PNG)
